myapp/backend/backend.py

```diff
@@ -175,8 +175,8 @@ def get_team_roster():
             filters['status'] = status
         if role is not None:
             filters['role'] = role
-        if position is not None:
-            filters['position'] = position
+        # if position is not None:
```

Comment:
Just curious what position would entail since it is commented out.

Reply:
I commented out since we will want to implement a filter by position in a future story. Position would be one of the following :  'Advisor', 'Director of Competitions', 'Lead', 'Lead Mentor', 'Member', 'Mentor', 'Parent', 'President', 'Secretary of Communications', 'Vice President'

Since it currently is not being used by the filter, I'll delete the line since it doesn't break anything!